backend/src/Service/Order/OrderService.php

```diff
@@ -65,10 +77,21 @@ public function getStoreOrders($userId): ?array
      * @param $id
      * @return OrdersResponse
      */
-    public function getSpecificOrderForStore($id): ?OrdersResponse
+    public function getSpecificOrderForStore(int $id): ?OrdersResponse
     {
         $order = $this->orderManager->getSpecificOrderForStore($id);
 
+        $order['images'] = $this->getImageParams($order['images'], $this->params.$order['images'], $this->params);
+
         return $this->autoMapping->map("array", OrdersResponse::class, $order);
     }
+    
+    public function getImageParams($imageURL, $image, $baseURL): array
```

Comment:
Also, this function we won't redefine it here as long as it already exists in UploadFileHelperService .